lib/include/jxl/encode.h

```diff
@@ -283,6 +283,12 @@ typedef enum {
    */
   JXL_ENC_FRAME_INDEX_BOX = 31,
 
+  /** Sets brotli encode effort for use in JPEG recompression and compressed
+   * metadata boxes (brob). Ranges from 0 (fastest) to 11 (slowest).
+   * Default: 9.
```

Comment:
I think the default should be -1, and chosen based on the encoder effort :)

Reply:
Makes sense. At least for jbrd we can do that. For the brob boxes, we don't really have an encoder effort, since that is part of the frame settings, so there is no encoder effort defined for these.

For brob boxes perhaps a sensible default is something lower than 9. I wanted to add this option because 9 was too slow for psd reconstruction boxes, which could be quite large in some cases so the brobbing was then actually a big bottleneck in the recompression process...